qemu/tests/cfg/usb.cfg

```diff
@@ -262,7 +262,7 @@
                 - usb_multi_times:
                     usb_repeat_times = 5000
         - usb_multi_disk:
-            only ehci
+            only usb-ehci
```

Comment:
One more question:
As you said, usb-ehci controller can not connect with low-speed usb device when machine type is rhel6.*.

how to solve it on rhel6 host?